static/js/actions/index.js

```diff
@@ -37,10 +41,20 @@ export const updateProfile = profile => ({
   type: UPDATE_PROFILE,
   payload: { profile }
 });
+export const openEducationForm = (level, index, education_id) => ({
```

Comment:
It seems like this should only take the `index`, `education_id` duplicates this purpose. Can you remove `education_id` from the reducer too?